drafts/WEP0002-slack-community-moderation-procedures.rst

```diff
@@ -0,0 +1,142 @@
+
+There's a certain amount of personal discretion that goes into removing posts, but when in doubt, the moderation team should generally refrain from removing posts. A lack of discretion in removing posts can negatively impact the community's trust in the moderation team.
+
+Sales-y content
+---------------
+Historically, the moderation team has struggled with the definition of "sales-y content" listed on the Slack guidelines. We currently define sales-y content as follows:
+
+* Direct pitches
+* Links without context
+* Brands
+
+Some further proposed definitions for the purpose of moderating are as follows:
+
+* **Unpublished market research** - being a helpful bunch, the Write the Docs community is never hesitant to weigh in on their experiences. However, market research that the community contributes to should also benefit the community. Therefore, we ask that by posting a survey and asking for responses, a user commits to allowing the community to access the results for free, and we reserve the right to decline or remove a survey if we don't think that it'll benefit the community sufficiently. We could also create a #surveys channel so users who aren't interested in surveys don't see them. (Academic surveys for a user's studies don't have to be held to the same standards, but a channel for surveys could also theoretically benefit users who are in school and are looking for information about the field.)
+* **Recruiter spam** - while it's not inherently bad for recruiters to find candidates within the Slack community, we want to make sure they don't detract from it in doing so. Spamming the community with job postings (and thereby distracting from posts from users who are affiliated with the postings) and pushy DMs give the impression that they're trying to mine the community for financial reasons, and that's not what we want. Because moderators can't monitor DMs, we depend on users to speak up when they happen in order to take action, and should communicate clearly that they can and should.
+* **Marketing posts in #intros** - the purpose of the #intros channel is to get to know new members of the community. It's often a difficult line to draw between a person and their work, but a general rule of thumb is that users shouldn't be looking for sales prospects in that channel. Posts including things like personal websites and social media links with invitations to connect are okay; however, writing on behalf of a business looking for potential clients would fall under sales-y content and thus would be subject to removal. At that point, a moderator can send the user a message explaining the removal, along with a warm invitation to post again without the sales angle.
```

Comment:
> a moderator can send the user a message explaining the removal, along with a warm invitation to post again without the sales angle.

I'd love to see some templated responses for common situation like this, along with the 3 strikes rule. It doesn't need to live in this document, but having a way to accumulate these templates over time will be useful.

Reply:
Just added a sample script! I think having everything in one place makes sense.

The only thing I'm still unsure of is whether I can format the block quote to look like ones between paragraphs rather than between bullet points, but either way, the content is there.